src/test/opentrackIO_parser/python/opentrackIO_parser.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
```

Comment:
Use same style as other python file naming i.e. opentrackio_parser.py

Reply:
Not sure what you're saying here. Get rid of the "shebang" ?

Reply:
I just mean rename file with lower case to match everything else